client/src/modules/list/components/ItemsList/ItemsList.jsx

```diff
@@ -41,24 +41,19 @@ class ItemsList extends Component {
   };
 
   voteForItem = item => () => {
-    const { _id, voterIds } = item;
+    const { _id, didCurrentUserVoted } = item;
```

Comment:
there is always one user so maybe didUserVote should be enough, I think there is no need to use Voted if you use did?